linux/tools.Dockerfile

```
@@ -71,5 +71,7 @@ RUN ln -s /usr/bin/node /usr/bin/nodejs
 # Add dotnet tools to PATH so users can install a tool using dotnet tools and can execute that command from any directory
 ENV PATH ~/.local/bin:~/bin:~/.dotnet/tools:$PATH
 
+ENV AZURE_CORE_SHOW_SECRETS_WARNING True
```

```suggestion
ENV AZURE_CLIENTS_SHOW_SECRETS_WARNING True
```
We may decide to change the env variable name. cc @isra-fel @dcaro for confirmation